core/src/tests/addresses_and_ports.cc

```diff
@@ -135,8 +85,8 @@ static bool test_sockets(int family, int port)
   bool result = true;
 
   if (family == 0) {
-    result = create_and_bind_v4socket(v4_fd, port)
-             && create_and_bind_v6socket(v6_fd, port);
+    result = create_and_bind_v4socket(v4_fd, port);
+    result &= create_and_bind_v6socket(v6_fd, port);
```

Comment:
Why this change? I guess to avoid short-cicuit evalutiont, right? However, &= is a bitwise AND and not logical AND.
For clarification it might make sense to introduce `bool result_v4` and `bool result_v6`and then have `result = restult_v4 && result_v6`

Reply:
good point